arex-storage-web-api/src/main/java/com/arextest/storage/service/QueryConfigService.java

```diff
@@ -75,6 +106,110 @@ public ScheduleReplayConfigurationResponse queryScheduleReplayConfiguration(Stri
+
+    List<ApplicationOperationConfiguration> operationConfigurationList =
+        applicationOperationConfigurationRepository.listBy(appId);
+    Map<String, ApplicationOperationConfiguration> operationConfigurationMap = operationConfigurationList.stream()
+        .collect(
+            Collectors.toMap(ApplicationOperationConfiguration::getId, operation -> operation));
+
+    Map<String, AppContract> contractMap = new HashMap<>();
+    List<AppContract> contracts = appContractRepository.queryAppContracts(appId);
+    if (CollectionUtils.isNotEmpty(contracts)) {
+      contractMap.putAll(
+          contracts.stream().collect(Collectors.toMap(AppContract::getId, contract -> contract)));
+    }
+
+    List<ComparisonExclusionsConfiguration> newConfigs = new ArrayList<>();
+    for (ComparisonExclusionsConfiguration config : configs) {
+      //dependency
+      if (config.getDependencyId() != null) {
+        AppContract contract = contractMap.get(config.getDependencyId());
+        if (contract != null) {
+          config.setOperationType(contract.getOperationType());
+          config.setOperationName(contract.getOperationName());
+          newConfigs.add(config);
+        }
+        continue;
+      }
+
+      ApplicationOperationConfiguration operationConfiguration = operationConfigurationMap.get(config.getOperationId());
+      if (operationConfiguration != null) {
+        config.setOperationName(operationConfiguration.getOperationName());
+        if (operationConfiguration.getOperationTypes().size() > 1) {
+          for (String operationType : operationConfiguration.getOperationTypes()) {
+            ComparisonExclusionsConfiguration newConfig = new ComparisonExclusionsConfiguration();
+            newConfig.setOperationName(config.getOperationName());
+            newConfig.setOperationType(operationType);
+            newConfig.setExclusions(config.getExclusions());
+            newConfigs.add(newConfig);
+          }
+        } else if (operationConfiguration.getOperationTypes().size() == 1) {
+          String operationType = operationConfiguration.getOperationTypes().iterator().next();
+          config.setOperationType(operationType);
+          newConfigs.add(config);
+        }
+      } else {
+        newConfigs.add(config);
+      }
+    }
+
+    Map<String, List<ComparisonExclusionsConfiguration>> groupByOperationNameAndType = newConfigs
+        .stream()
+        .collect(Collectors.groupingBy(
+            config -> config.getOperationName() + config.getOperationType()));
+    List<ConfigComparisonExclusions> result = new ArrayList<>();
+    for (Entry<String, List<ComparisonExclusionsConfiguration>> entry : groupByOperationNameAndType
+        .entrySet()) {
+      List<ComparisonExclusionsConfiguration> configList = entry.getValue();
+      if (CollectionUtils.isEmpty(configList)) {
+        continue;
+      }
+      ConfigComparisonExclusions vo = new ConfigComparisonExclusions();
```

Comment:
Try not to name it with `vo` as much as possible, replace `configComparisonExclusions`